### PR TITLE
Blog - PyTest Fixtures

### PR DESCRIPTION
**Pull request by msali123**
Done with all checks.

### CODE REVIEW

**Comment by msali123** [comment]
[Link](https://642a294d12437a7e83873029--ephemeral-narwhal-3ac3b2.netlify.app/blog/pytest-fixtures/)

@jalletto @balapriyac

File: blog/_posts/2023-05-14-pytest-fixtures.md [hunk 1]
```diff
@@ -0,0 +1,554 @@
+---
+title: "PyTest Fixtures"
```
Comment (balapriyac) [inline blog/_posts/2023-05-14-pytest-fixtures.md:2]:
```suggestion
title: "Getting Started With PyTest Fixtures"
```

**Review by balapriyac — CHANGES_REQUESTED** [review]
Great work, Sameer!

**Comment by msali123** [comment]
> Great work, Sameer!

Thank you @balapriyac 

[Preview Link](https://642a9f85b23389688177a161--ephemeral-narwhal-3ac3b2.netlify.app/blog/pytest-fixtures/)

**Comment by balapriyac** [comment]
@jalletto this PR is ready to merge, too.

**Review by jalletto — APPROVED** [review]

**Pull request merged**